static/src/js/components/CustomCheckboxWidget/__tests__/CustomCheckboxWidget.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import userEvent from '@testing-library/user-event'
+import { render, screen } from '@testing-library/react'
+
+import CustomWidgetWrapper from '../../CustomWidgetWrapper/CustomWidgetWrapper'
+import CustomCheckboxWidget from '../CustomCheckboxWidget'
+
+vi.mock('../../CustomWidgetWrapper/CustomWidgetWrapper')
+
+const setup = (overrideProps = {}) => {
+  const props = {
+    id: 'mock-id',
+    label: 'Test Field',
+    onChange: vi.fn(),
+    registry: {
+      formContext: {
+        focusField: '',
+        setFocusField: vi.fn()
+      }
+    },
+    required: true,
+    schema: {
+      description: 'Test Description'
+    },
+    uiSchema: {},
+    value: false,
+    ...overrideProps
+  }
+
+  render(
+    <CustomCheckboxWidget {...props} />
+  )
+
+  return {
+    props,
+    user: userEvent.setup()
+  }
+}
+
+beforeEach(() => {
+  CustomWidgetWrapper.mockImplementation(
+    vi.importActual('../../CustomWidgetWrapper/CustomWidgetWrapper').default
```

Comment:
If every test needs the actual CustomWidgetWrapper then include this in the `vi.mock()` call. But you also aren't doing any assertions on CustomWidgetWrapper, so do you even need to mock it at all?